web/src/emoji_picker.ts

```diff
@@ -59,6 +59,11 @@ let edit_message_id: number | null = null;
 let current_message_id: number | null = null;
 
 const EMOJI_CATEGORIES = [
+    {
+        name: "Frequently used",
+        icon: "fa-refresh",
+        translated: $t({defaultMessage: "Frequently used"}),
+    },
     {name: "Suggested", icon: "fa-star-o", translated: $t({defaultMessage: "Suggested"})},
```

Comment:
I think rather than having two separate categories, I think we should just have 1 category, called "Frequently used". It can have the union of these two data sources: We offer up to 12 emoji, preferring those that we know are actually used over those in our set of 6 "built-in" frequently used emoji.